pyaedt/modeler/advanced_cad/stackup_3d.py

```diff
@@ -3259,7 +3374,7 @@ class MachineLearningPatch(Patch, object):
     frequency : float, None
         The patch frequency, it is used in prediction formulas. If it is None, the patch frequency will be that of the
         layer or of the stackup. From 0.1 to 10 GHz.
-    patch_width : float
+    dx : float
         The patch width. From O.5 to 1.5 of the optimal width value : c0 * 1000/(2 * f * sqrt((er  + 1)/2))
     signal_layer : :class:`pyaedt.modeler.stackup_3d.Layer3D`
         The signal layer where the patch will be drawn.
```

Comment:
```suggestion
        Patch width, which is from 0.5 to 1.5 of the optimal width value: c0 * 1000/(2 * f * sqrt((er  + 1)/2)).
    signal_layer : :class:`pyaedt.modeler.stackup_3d.Layer3D`
        Signal layer where the patch is to be drawn.
```
The description here for ``patch_width`` is also more detailed than the previous occurrence. Should they be the same?